application-flashmessages-ui/src/main/resources/Flash/FlashSheet.xml

```diff
@@ -39,67 +39,97 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{include reference='Flash.Macros' /}}
+  <content>{{include reference="Flash.Macros"/}}
+
+{{velocity output="false"}}
+#macro (displayFlashProperty $obj $propName)
+  #set ($class = $obj.xWikiClass)
+  #set ($prettyName = $class.get($propName).translatedPrettyName)
```

Comment:
There's no need for this, you should rather use `$doc.displayPrettyName($propName)`. See https://jira.xwiki.org/browse/XWIKI-15563.